src/frontend/ducks/ledgers.ts

```diff
@@ -102,6 +102,11 @@ export const startLedgerStreamingAction = createAsyncThunk<
         .stream({
           onmessage: (ledger: LedgerRecord) => {
             const { lastLedgerRecords } = ledgersSelector(getState());
+
+            if (!lastLedgerRecords[0]) {
```

Comment:
just curious: is it possible for this array to have a length but have no record (or null) at index 0? I'm assuming no

Reply:
mainly wondering why you chose to check for this instead of `lastLedgerRecords.length`

Reply:
Good point, we could be checking just for the array length here, but I went with the `lastLedgerRecords[0]` because it's used below in `lastLedgerRecords[0].closedAt`. Do you think it would more clear to check for the array length?